config/services.yaml

```diff
@@ -38,6 +43,11 @@ services:
             $userWithAssignmentsCacheTtl: '%app.cache.ttl.get_user_with_assignments%'
             $ltiVersion: '%app.lti.version%'
             $ltiRegistrationId: '%app.lti1p3.registration_id%'
+            $privateKeyPath: '%app.jwt.private_key_path%'
```

Comment:
Let's be more specific with the bindings, `$privatekeyPath` can be used in multiple different scenarios. Let's use a more specific name, OR preferably let's autowire it directly on the service which is using it